### ISSUE TITLE
Error using initialization fields

### ISSUE THREAD

**Issue by 0c370t** [muhlba91/pulumi-proxmoxve#24]
When attempting to create a Cloud Template VM using this provider, I am presented with this error:
```
  proxmoxve:VM:VirtualMachine (Debian Cloud Template Template Virtual Machine):
    error: 1 error occurred:
        * creating urn:pulumi:infra::homelab-infrastructure::HomeLab:Proxmox:CloudImageTemplate$proxmoxve:VM/virtualMachine:VirtualMachine::Debian Cloud Template Template Virtual Machine: 1 error occurred:
        * Image resized.
    unable to parse volume filename '10000000/vm-10000000-disk-0.qcow2.qcow2'
```

I've put the relevant code here:
https://gist.github.com/0c370t/1fbaecbbbb32d18d7a9207e07c8b1e5b

Related:
https://github.com/bpg/terraform-provider-proxmox/issues/203

**Comment by muhlba91** [comment]
thanks for the information - once the upstream issue is addressed, i can update this provider.

**Comment by 0c370t** [comment]
Looks like this has been fixed upstream

**Comment by muhlba91** [comment]
do you happen to know if this was released in 0.13.0 upstream or will this be in the next release?

yesterday i released a new version using 0.13.0 upstream.

**Comment by 0c370t** [comment]
https://github.com/bpg/terraform-provider-proxmox/pull/241

Looks like it was included
